test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -904,7 +901,7 @@ DEFINE_PROTO_FUZZER(const api_fuzzer::Msg& msg) {
         bool ok = true;
         if (g_channel == nullptr) ok = false;
         Call* parent_call = ActiveCall();
-        if (parent_call != nullptr && parent_call->type() == CallType::CLIENT) {
+        if (parent_call != nullptr && parent_call->type() != CallType::SERVER) {
```

Comment:
Could you add a comment why for CallType::SERVER, the current active call becomes a parent ?

Reply:
done